### ISSUE TITLE
Don't disable langpack versions when two uploads happen around the same time

### ISSUE THREAD

**Issue by jcristau** [mozilla/addons-server#21096]
We've had two instances lately of langpack uploads failing because a firefox dot release and firefox beta builds were happening around the same time, and the release langpack upload for one locale was set to "disabled" instead of getting processed and signed independently of the beta one.  Firefox release automation expects to get signed langpacks for each version (they set min/max version that are disjoint, so we need both).

In bugzilla:
https://bugzilla.mozilla.org/show_bug.cgi?id=1837547
https://bugzilla.mozilla.org/show_bug.cgi?id=1848872

**Comment by diox** [comment]
Relevant code: https://github.com/mozilla/addons-server/blob/4bfacd4fbd71c4c24a49d8ef3ae71b9900017770/src/olympia/versions/models.py#L910-L927

We do this for all add-ons on the listed channel so that there is only one version awaiting review at the same time. This has been there since the beginning of time, but langpacks are special...

One problem is, adding a check there for langpacks might not be enough. I'm not sure how auto-approval will behave with 2 listed versions awaiting review...

**Comment by diox** [comment]
> One problem is, adding a check there for langpacks might not be enough. I'm not sure how auto-approval will behave with 2 listed versions awaiting review...

Actually... we should test it further, but I think that would just work. `current_version` will be unreliable (the last uploaded version that has been approved will win), but that's acceptable for langpacks.

**Comment by gabrielBusta** [comment]
@diox could an exception be added for the langpacks to the Relevant code?

**Comment by diox** [comment]
That's what this issue is all about, yes. We're looking into it.

**Comment by ioanarusiczki** [comment]
@diox 

I tested on AMO Dev, uploading new versions one after the other. I did not see any listed versions being disabled for a lang pack. 

These are my uploads on dev 
https://addons-dev.allizom.org/en-US/developers/addon/language-pack-test2/versions
https://addons-dev.allizom.org/en-US/developers/addon/nederlands-dutch-on-dev/versions

Extensions and static themes are working as before: a new upload disables the previous version in awaiting review state on the listed channel.
